app-text/jabref/jabref-2.10-r7.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+JAVA_PKG_IUSE="doc"
+JAVA_ANT_REWRITE_CLASSPATH="true"
+EANT_BUILD_TARGET="jars"
+EANT_DOC_TARGET="docs"
+
+inherit desktop java-pkg-2 java-ant-2 xdg-utils
+
+MY_PV="${PV/_beta/b}"
+
+DESCRIPTION="Java GUI for managing BibTeX and other bibliographies"
+HOMEPAGE="https://www.jabref.org/"
+SRC_URI="mirror://sourceforge/${PN}/JabRef-${MY_PV}-src.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+
+CP_DEPEND="
+	dev-java/antlr:0
+	dev-java/antlr:3
+	dev-java/fontbox:1.7
+	dev-java/jempbox:1.7
+	dev-java/log4j:0
+	dev-java/spin:0
+	dev-java/microba:0
+	>=dev-java/glazedlists-1.8.0:0"
+
+DEPEND="
+	virtual/jdk:1.8
+	${CP_DEPEND}"
+
+RDEPEND="
+	virtual/jre:1.8
```

Comment:
please add a comment why we need to restrict to `:1.8`

Reply:
done!